geps/gep-1911.md

```diff
@@ -0,0 +1,190 @@
+# GEP-1911: Backend Protocol Selection
+
+* Issue: [#1911](https://github.com/kubernetes-sigs/gateway-api/issues/1911)
+* Status: Provisional
+
+(See status definitions [here](overview.md#status).)
+
+## TLDR
+
+Not all implementation support automatic protocol selection. Even in some cases protocols are disabled without an explicit opt-in (ie. websockets). Thus application developers need the ability specify the protocol(s) that their application supports.
+
+## Goals
+
+- Support protocols that can have a Gateway '*Route' resource as a frontend
+- Consistent way of declaring backend protocols using a meta-resource
+- Support backends with multiple protocols on the same port (ie. h2c/http1)
+- Extensible - additional protocol specific fields should be easy to add in the future
+
+## Non-Goals
+
+- Backend TLS (covered in [GEP-1897](https://github.com/kubernetes-sigs/gateway-api/issues/1897)
+
+## Introduction
+
+## API
+
+We propose adding a new meta-resource [GEP-713](geps/gep-713.md) called `BackendProtocol`.
```

Comment:
Is #713 the right GEP?

Reply:
This is open for discussion. Technically since I'm using a `targetRef` it could be _anything_. But if we want to scope this to just `Service` then we could simplify this.

As is,  _we_ could target a multi cluster ServiceImport - but I'm not to familiar with that. cc @robscott

Reply:
Re-wrote the GEP so it's not a meta-resource anymore